tests/Command/CommandTest.php

```diff
@@ -99,4 +111,28 @@ public function setting_unknown_defaults_throws_an_exception()
             'doesnotexist' => '0',
         ]);
     }
+
+    /**
+     * @test
+     */
+    public function arguments_that_do_not_map_to_defaults_are_not_added()
+    {
+        $this->application->command('greet [name]', [new GreetCommand, 'greet']);
```

Comment:
The point of the test was to verify that it was _not_ throwing an exception about options / arguments not existing (the parameter is called `$times` in the `greet` method which does not exist as an option or argument). I should've written a comment to be more clear.

That all being said I know there is definitely a way to be more explicit in the test. Do you have a recommendation?